examples/isentropic_example.py

```diff
@@ -0,0 +1,227 @@
+# Copyright (c) 2008-2016 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""
+==============================
+Isentropic Analysis with MetPy
+==============================
+
+The MetPy function *mcalc.isentropic_interpolation* allows for isentropic analysis from model
```

Comment:
Use double backticks rather than * to make the reference--that should make it actually link to it.